src/WebJobs.Script.Grpc/Channel/GrpcWorkerChannel.cs

```diff
@@ -553,6 +554,19 @@ internal FunctionLoadRequest GetFunctionLoadRequest(FunctionMetadata metadata, M
 
                 request.Metadata.Bindings.Add(binding.Name, bindingInfo);
             }
+
+            foreach (var property in metadata.Properties)
+            {
+                // Only add worker properties
```

Comment:
Why only worker properties? What is the source of the properties you're trying to flow?

Reply:
Why not just include all metadata properties - why do we need filtering?

Reply:
I think the idea is to be deliberate on what properties the worker wants to pass to the load request. The worker would be adding the prefix `worker.` to the properties it wants to pass on the other workers via load request. 

cc: @madelinegordon , @fabiocav, was there another reason to only include properties with `worker.` prefix?

Reply:
I think the reason it's only for worker properties is that the requirement in #8302 is from the portal team, for the purpose of being able to switch the view to other languages. But I'm not 100% sure that's the reason.

Reply:
I don't think we need to filter those, but I believe the original ask was to have the prefix removed before sending it back. As-is, this is something the worker needs to be aware of, as if the worker adds a property named `foo` it would roundtrip with the name `worker.foo`

Reply:
@fabiocav I am a little confused by your comment. We are not adding any prefix to the properties. Just roundtripping the properties one with prefix included.

In any case based on the comments it looks like we don't really need to filter the properties with prefix. I had a chat with @gavin-aguiar, looks like they were not expecting the prefix either. I am going to make a change so that we round-trip all properties As-Is.